### PR TITLE
Add envfile input format

### PR DESCRIPTION
**Pull request by mattrobenolt**
Fixes GH-38

### CODE REVIEW

**Comment by razvanphp** [comment]
Please add this to the `README.md`, really helps new users, maybe also specifying `environ()` trick too.

Thank you!

File: jinja2cli/cli.py [hunk 1]
```diff
@@ -146,6 +150,23 @@ def _load_xml():
...
 
 
+def _load_env():
+    def _parse_env(data):
+        """
+        Parse an envfile format of key=value pairs that are newline separated
+        """
+        dict_ = {}
+        for line in data.splitlines():
+            line = line.strip()
```
Comment (razvanphp) [inline jinja2cli/cli.py]: I think `strip()` has to be called after `split(=)`, so both key & value have the spaces removed.

Comment (mattrobenolt) [inline jinja2cli/cli.py]: That's not how envfile format works. If you used that value, you'd be getting a key of "foo " and a value of " blah blah blah", and would be intended behavior for environment variables. See my reference material in the issue.

Comment (mattrobenolt) [inline jinja2cli/cli.py]: I also changed this to `lstrip` since it should only be stripping preceding spaces, not trailing spaces.

Comment (mattrobenolt) [inline jinja2cli/cli.py]: Also based on what you originally said "dotenv", there are implementation of that that wouldn't cover what you're saying either:
https://www.npmjs.com/package/dotenv#rules for example which is much closer to what I wrote here. I'm matching Docker's behavior.

**Pull request merged**